packages/voice/src/util/Secretbox.ts

```diff
@@ -5,6 +5,30 @@ interface Methods {
 }
 
 const libs = {
+	'sodium-native': (sodium: any): Methods => ({
+		open: (buffer: Buffer, nonce: Buffer, secretKey: Uint8Array) => {
+			if (Boolean(buffer)) {
```

Comment:
Isn't this always going to return true?

Reply:
there was a test that failed because it doesn't pass a buffer :/

Reply:
Yeah there's only the one secret box test, which fails without the null check, so I opted to respect the test. I realize that there probably isn't going to be a situation where a buffer isn't passed during actual run, so I'll change it if you want. At that point I figure an empty buffer should be passed in the test though.

Reply:
Out of curiosity, what test are you talking about? 👀

Reply:
[this one](https://github.com/discordjs/discord.js/blob/main/packages/voice/__tests__/Secretbox.test.ts)

Reply:
I guess just change this to `if (buffer)`? 🤷 I don't really know what is the expected output when giving wrong inputs..

Reply:
```
@discordjs/voice:format: \discord.js\packages\voice\src\util\Secretbox.ts
@discordjs/voice:format:   10:8  error  Unnecessary conditional, value is always truthy  @typescript-eslint/no-unnecessary-condition
```
🤷‍♂️
I figured `!!buffer` would look nicer than another 'disable next line' comment, but the linter changed it to bool(buffer)
I'll just do another comment